uportal-war/src/main/resources/properties/contexts/schedulerContext.xml

```diff
@@ -51,25 +51,26 @@
         <property name="daemon" value="true" />
         <property name="rejectedExecutionHandler" ref="loggingAbortHandler" />
     </bean>
-    
+
     <task:scheduled-tasks scheduler="uPortalTaskScheduler">
         <!--
          | Prime numbers are used so that executions will be distributed over time. Use this site for
          | finding primes: http://www.numberempire.com/primenumbers.php Generate a 3 or 4 digit prime
          | and then append enough 0s to get the duration scale required.
          +-->
-         
+
         <!-- local tasks -->
         <task:scheduled ref="portalEventDaoQueuingEventHandler" method="flush" fixed-delay="997"/> <!-- ~1 second period -->
         <task:scheduled ref="cacheManagerExpiredElementEvictor" method="evictExpiredElements" fixed-delay="61900"/> <!-- ~1 minute period -->
-        
+        <task:scheduled ref="portletPermissionsCachePrimer" method="primeCache" fixed-delay="299993"/> <!-- just under 5 minute period;  should be shorter than the TTL for the cache named 'org.jasig.portal.security.provider.AnyUnblockedGrantPermissionPolicy.HAS_UNBLOCKED_GRANT' -->
```

Comment:
I suggest a fixed-delay that has a bit more buffer than 7ms.  I would expect on a busy portal it is likely that the scheduled task will not execute in time and a request thread (or set of them) will get hit with the permissions evaluation time.  I would have gone at least 1 sec shorter.